Create-Login-and-Register-System-with-C#-Winforms/index.md

```diff
@@ -0,0 +1,430 @@
+### Introduction
+
+C# or C-sharp is an object-oriented programming language developed by Microsoft that runs on a framework called .Net framework. The language has been used to develop many kinds of applications including web applications, mobile applications, computer games, desktop applications, database applications, virtual reality applications, and many others.
+
+### Prerequisites
+
+* A code editor e.g [Visual Studio](https://visualstudio.microsoft.com/vs/).
+* Knowledge of C#, SQL.
+* Running relational database management in this case we will use [MySQL](https://www.mysql.com/)
+* Visual Studio Knowledge
+
+### Step I: Create a database and table with required columns
+
+``` sql
+
+Create database userdata;
+```
+
+![DATABASE CREATION]((/Create-Login-and-Register-System-with-C#-Winforms/create-db-cmd.png))
+
```

Comment:
The image attached uses MariaDB instead of MySQL, just stick to your requirement stated above. Even though MariaDB is a fork of MySQL, these two database management systems are still quite different.

Reply:
sorted